### PR TITLE
Monasca notif update

### PR DESCRIPTION
**Pull request by dougszumski**

### CODE REVIEW

**Comment by dougszumski** [comment]
Depends on K-A support.

File: etc/kayobe/kolla/config/monasca/notification.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[notification_types]
+enabled = slack
+
+[slack_notifier]
+message_template: "/etc/monasca/slack_template.j2"
+timeout: 10
+ca_certs: "/etc/ssl/certs/ca-bundle.crt"
+insecure: False
```
Comment (markgoddard) [inline etc/kayobe/kolla/config/monasca/notification.conf]: should these be key = value INI format?

Comment (dougszumski) [reply etc/kayobe/kolla/config/monasca/notification.conf]: Oops, for consistency yes, although it seems to work fine.

**Comment by dougszumski** [comment]
Ready for merge when kolla-ansible stackhpc/6.1.0.4 is tagged and contains the change which supports this.

**Comment by dougszumski** [comment]
@markgoddard would you like to see anything more addressed in this?

**Review by markgoddard — APPROVED** [review]

**Pull request merged**